emcc.py

```diff
@@ -476,6 +476,23 @@ def validate_arg_level(level_string, max_level, err_msg):
         raise Exception(err_msg)
       return level
 
+    def has_fixed_language_mode(args):
```

Comment:
looks like this would parse all args each time it is called? could it instead be done once beforehand?